console/src/components/PresetItem/PresetItem.js

```diff
@@ -0,0 +1,56 @@
+import {ItemBase} from '@enact/agate/Item';
+import {LabeledItemBase} from '@enact/agate/LabeledItem';
+
+import {forward, handle} from '@enact/core/handle';
+import Spottable from '@enact/spotlight/Spottable';
+import Pure from '@enact/ui/internal/Pure';
+import SlotItem from '@enact/ui/SlotItem';
+import Touchable from '@enact/ui/Touchable';
+import compose from 'ramda/src/compose';
+import kind from '@enact/core/kind';
+import React from 'react';
+
+import componentCss from './PresetItem.less';
+
+const PresetItemBase = kind({
+	name: 'PresetItem',
+
+	propTypes:{},
+
+	styles: {
+		css: componentCss,
+		className: 'presetItem'
+	},
+
+	handlers: {
+		onClick: (ev, props) => {
```

Comment:
Could use `handle` + `adaptEvent` here. Perhaps we should add a "shortcut" in `core/handle` for the common sequence. Seems like it comes up relatively often.

```
const forwardCustom = (name, adapter) => adapter ? adaptEvent(adapter, forward(name)) : forward(name);
// or
const handleAndForward = (name, adapter) => handle(adapter ? adaptEvent(adapter, forward(name)) : forward(name));
```